### PR TITLE
Update information on GD Advanced Protection Program

### PR DESCRIPTION
**Pull request by Cassie205**

### CODE REVIEW

File: protocols/googledrive.md [hunk 1]
```diff
@@ -22,9 +22,11 @@ Using prior versions to 4.12.0 (Mountain Duck) or 8.4.0 (Cyberduck), you might *
 
 Google Drive uses OAuth 2 to allow Cyberduck to access your files on Google Drive. Choose your account email as the username in the bookmark and choose *Allow* on the website opened in your default web browser to allow Cyberduck to *View and manage the files in your Google Drive*. The authorization code will be submitted to Cyberduck automatically. Subsequent connections will not require authorization, unless the refresh token itself is expired due to inactivity.
 
-```{attention}
-Using *[Advanced Protection Program](https://support.google.com/accounts/answer/7539956#non-goog_apps&zippy=%2Ccan-i-use-non-google-apps-services-or-apps-script-with-advanced-protection)* will cause the OAuth login flow to fail. 
-```
+#### Advanced Protection Program
+
+Using *[Advanced Protection Program](https://support.google.com/accounts/answer/7539956#non-goog_apps&zippy=%2Ccan-i-use-non-google-apps-services-or-apps-script-with-advanced-protection)* will cause the OAuth login flow to fail. Mostly with the error message: `400 admin_policy_enforced`.
```
Comment (dkocher) [inline protocols/googledrive.md:27]:
```
Using *[Advanced Protection Program](https://support.google.com/accounts/answer/7539956#non-goog_apps&zippy=%2Ccan-i-use-non-google-apps-services-or-apps-script-with-advanced-protection)* will cause the OAuth login flow to fail with the error message `400 admin_policy_enforced`.
```

File: protocols/googledrive.md [hunk 2]
```diff
@@ -22,9 +22,11 @@ Using prior versions to 4.12.0 (Mountain Duck) or 8.4.0 (Cyberduck), you might *
...
+
+You can resolve the error, please reach out to your Google Workspace admin to grant access to Mountain Duck or Cyberduck.
```
Comment (dkocher) [inline protocols/googledrive.md:29]:
```suggestion
Reach out to your Google Workspace admin to grant access to Mountain Duck or Cyberduck.
```

**Review by dkocher — CHANGES_REQUESTED** [review]
I was not aware resolving the error is possible. Where is this documented from Google?

File: protocols/googledrive.md (hunk 1 above)
Comment (Cassie205) [reply protocols/googledrive.md:27]: https://support.google.com/accounts/answer/12917337?hl=en#400admin&zippy=%2Cadmin-policy-enforced

Based on my understanding the Admin can choose to allow or deny log in attempts by third party applications.

**Review by dkocher — APPROVED** [review]

**Pull request merged**